spec/models/onboard_judging_spec.rb

```diff
@@ -3,3 +3,25 @@
 RSpec.describe OnboardJudging, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
 end
+
+describe OnboardJudging do
+  let!(:onboard_judging) { create :onboard_judging }
+  describe 'set_score' do
+    it 'private method is run' do
```

Comment:
Well, here you're just duplicating the content of the private method. Let's work tomorrow on a better way of doing this.